components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -475,13 +475,7 @@ @implementation MDCLegacyInkLayer
 - (instancetype)init {
   self = [super init];
   if (self) {
-    self.masksToBounds = YES;
```

Comment:
it looks like we have lost this masksToBounds = YES with this change.

Reply:
Thanks for catching. Done